### ISSUE TITLE
[REGRESSION] URL rewritting completly broken with gbf.wiki and other wikis

### ISSUE THREAD

**Issue by kelson42** [openzim/mwoffliner#1017]
With 1.9.12:
```
mwoffliner --mwUrl=https://gbf.wiki/ --adminEmail=kelson@kiwix.org --filenamePrefix=granbluefantasy_en_all --format=nopic --format=novid:maxi --mwWikiPath=index.php --mwModulePath=load.php --mwApiPath=api.php --useCache --verbose
```

All the internal links are broken.

**Comment by kelson42** [comment]
See also #1017

**Comment by kelson42** [comment]
This ZIM file is quite old... but regenerting it suffers from more massive problem, so difficult to say anything for now.

**Comment by bakshiutkarsha** [comment]
@kelson42  This issue doesn't seems to exists now. I think changes done by @midik in past which revolved around this ticket seems to have solve this. :)

**Comment by kelson42** [comment]
@bakshiutkarsha It is true, but it seems that the URL in subpages are still completly broken, maybe the same as #1094

**Comment by kelson42** [comment]
Closing, original bug is fixed and rest probably a duplicate of #1094.
